docs/source/developer/interactives.rst

```diff
@@ -174,7 +185,8 @@ Items of interest from this diagram:
       <!-- The JavaScript of your interactive goes here -->
     {% endblock js %}
 
-  The first line ``{% extends interactive_mode_template %}`` is required for all interactives.
+  The first line ``{% extends interactive_mode_template %}`` is required for all interactives. The second line ``{% load i18n %}`` is needed if you are translating any text in your template.
```

Comment:
Change to 'if you have translatable text in your template'

Reply:
Nice :)